File: Generated_Codes_Check/Results/java/140.java
/*
*/
import java.util.*;

public class Solution {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        while(t-- > 0){
            int k = sc.nextInt();
            int[] arr = new int[k];
            for(int i = 0; i < k; i++){
                int n = sc.nextInt();
                int temp = 0;
                for(int j = 0; j < n; j++){
                    int x = sc.nextInt();
                    temp += x;
                }
                if(temp % 2 == 0){
                    arr[i] = 1;
                }
            }
            if(arr[0] == 0){
                System.out.println("DON'T PLAY");
            }
            else{
                boolean ans = false;
                for(int i = 0; i < k; i++){
                    if(arr[i] == 0){
                        ans = true;
                    }
                }
                if(!ans){
                    System.out.println("DON'T PLAY");
                }
                else{
                    System.out.println("ODD");
                }
            }
        }
    }
}

File: Scripts/generated_codes/python/codechef_responses/140.py


*/

import java.util.*;
import java.io.*;
class TestClass {
    public static void main(String args[] ) throws Exception {
        int t,k,n,i,j,count,count1,count2,count3;
        Scanner sc=new Scanner(System.in);
        t=sc.nextInt();
        while(t>0)
        {
          k=sc.nextInt();
          count1=0;count2=0;
          while(k>0)
          {
              n=sc.nextInt();
              count=0;
              while(n>0)
              {
                  if(sc.nextInt()%2==0)
                  count++;
                  n--;
              }
              if(count==0)
              count2++;
              else if(count>0)
              count1++;
              k--;
          }
          if(count1>0 && count2==0)
          {
              System.out.println("EVEN");
          }
          else if(count2>0 && count1==0)
          {
              System.out.println("ODD");
          }
          else
          {
              System.out.println("DON'T PLAY");
          }
          t--;
        }
    }
}